### DISCUSSION TITLE
Can I disable "Visual Search" Icon?

### DISCUSSION THREAD

**Discussion by cottonspace** [fengyuanchen/viewerjs#626] [Q&A] [answered]
When I use Microsoft Edge, I can see a "Visual Search" icon in the top right corner of the image. I would like to not display this, is it possible? Possibly it is a feature of Microsoft Bing.

![image](https://github.com/fengyuanchen/viewerjs/assets/21207463/99e65025-6853-4d16-a9dd-cd7aafe60d61)

Note) I'm using the viewer.js version 1.10.

**Comment by fengyuanchen** [comment]
Extactly it is a feature of Microsoft Edge browser.

**Reply by cottonspace** [reply]
Thank you for your reply. I'm thinking hide the Visual Search Icon using CSS or something like that.

**Reply by sujan2791** [reply, marked as answer]
You can add css property "pointer-events: none" on that given image I hope your issue may resolved.

**Comment by fengyuanchen** [comment]
Also, it may be an additional button injected by some browser extensions you installed.
